### ISSUE TITLE
Add endpoint to query count milestone past cone information

### ISSUE THREAD

**Issue by luca-moser** [iotaledger/inx-chronicle#635]
This is an endpoint which is wanted from the explorer to be able to display the # of blocks and txs occurring within a particular milestone's past cone via https://github.com/orgs/iotaledger/projects/30/views/5.

A draft API response would return following data:
```
{
  "blocks": 100,
  "perPayloadType":{
    "noPayload": 95,
    "txsConfirmed": 5,
    "txsConflicting": 2,
    "taggedData": 1,
    "milestone": 1
  }
}
```

where `"blocks"`  is the # of blocks in the milestone's past cone
and `perPayloadType` the # of payload types occurring in the past cone.

There is a split between confirmed and conflicting transactions as to portray easier network health.

Since this call should be quick, it is better pre-computed when Chronicle consumes a milestone's past cone.

We would expect this information to be available under a `/api/explorer/v2/ledger/stats/by-milestone/{milestoneId}"` endpoint but are of course open to discuss specifics.

**Comment by grtlr** [comment]
The reason for this request is the following feature in the Explorer: https://github.com/iotaledger/explorer/issues/483

**Comment by grtlr** [comment]
Closed by #637.
